### PR TITLE
Use DNS SRV record in Thanos Ruler configuration for querier endpoint

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: jsonnet/thanos-ruler.jsonnet [hunk 1]
```diff
@@ -181,7 +181,7 @@ local thanosRulerRules =
                 server_name: 'thanos-querier.openshift-monitoring.svc',
               },
             },
-            static_configs: ['thanos-querier.openshift-monitoring.svc:9091'],
+            static_configs: ['dnssrv+_web._tcp.thanos-querier.openshift-monitoring.svc.cluster.local'],
```
Comment (lilic) [inline jsonnet/thanos-ruler.jsonnet:184]: hmm interesting, after discussing with @brancz and Bartek we said we didn't need this last time.

Comment (s-urbaniak) [reply jsonnet/thanos-ruler.jsonnet:184]: ping @bwplotka can you help in finding the correct setting?

**Comment by simonpasquier** [comment]
After offline discussion, it's been agreed that it isn't worth the change. If we feel that using SRV records (`_svc._tcp.host`) is more readable than A records + port (`host:port`) then we should switch to SRV everywhere we can.

**Pull request closed without merging**